tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json

```diff
@@ -380,6 +379,7 @@
     "onnx/node/generated/test_sce_none_weights_expanded",
     "onnx/node/generated/test_sce_none_weights_log_prob",
     "onnx/node/generated/test_sce_none_weights_log_prob_expanded",
+    "onnx/node/generated/test_shape_clip_start",
```

Comment:
This shape test fails on top of main too, so I don't think the failure is related to the integrate. There is an issue filed in torch-mlir for this test here: https://github.com/llvm/torch-mlir/issues/3841.